src/Controls/ResourceGrid.cs

```diff
@@ -108,9 +110,10 @@ private void ApplyConditionalFormatting(DataGridViewRow r)
                 r.DefaultCellStyle.ForeColor = dataGridView1.DefaultCellStyle.ForeColor;
             }
 
-            if (r == dataGridView1.Rows[RowCount - 1])
+            if (r == dataGridView1.Rows[RowCount - 1] && string.IsNullOrEmpty(r.Cells[colNameKey].Value?.ToString()))

```

Comment:
Why is the new check necessary? `RowCount - 1` are the comments and they shouldn't be highlighted.

Reply:
This is added because without this when translates the last line doesn't apply color format